services/web/server/tests/unit/with_dbs/03/folders/test_folders.py

```diff
@@ -0,0 +1,5 @@
+# pylint: disable=redefined-outer-name
```

Comment:
interesting tests ;-) (i am aware they are skeleton)

Reply:
THOUGHT: it would be interesting to have "default" tests in `pytest_simcore` to check some generic invariants . This way we cover the basics "for free"